vmm/src/cpu.rs

```diff
@@ -291,7 +291,6 @@ pub struct Vcpu {
     #[cfg(target_arch = "aarch64")]
     mpidr: u64,
     saved_state: Option<CpuState>,
-    configured: bool,
```

Comment:
I removed the `configured`. Because I think checking the lock is the same thing, we don't need to maintain a state.